core/src/main/java/org/apache/carbondata/core/datamap/DataMapStoreManager.java

```diff
@@ -163,6 +165,17 @@ private DataMapStoreManager() {
     return provider.retrieveAllSchemas();
   }
 
+  /**
+   * Return first match of the specified index name in table
+   *
+   */
+  public Optional<DataMapSchema> getIndexInTable(CarbonTable carbonTable, String indexName)
```

Comment:
DataMapSchema need to change to IndexSchema ?

Reply:
Yes, it will be changed in a new pr.